server/models/Books.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require("mongoose");
+mongoose.Promise = global.Promise;
+
+const Schema = mongoose.Schema;
+
+const novelSchema = new mongoose.Schema ({
+  bookType: String,  //types of books: novel, shortStory, inspiredStory, article
+  bookTitle: String,
+  bookChapter: String,  //can be chapter name or chapter number
+  bookPath: String
+},
+  {
+    collection: "novels"
+  })
+
+// const novelSchema = new mongoose.Schema({
+//   titleID: String,
+//   title: String,
+//   chapterName: String,
+//   file_title: String,
+//   file_url: String,
+// })
+// //
+// const shortsSchema = new mongoose.Schema ({
+//   shortsName: String,
+//   file_title: String,
+//   file_url: String,
+//   slug: String,
+// })
+// //
+// const storiesArticlesSchema = new mongoose.Schema ({
+//   type: String,
+//   title: String,
+//   text: String,
+//   slug: String,
+// })
+
+module.exports = mongoose.model("Novel", novelSchema);
+// module.exports = mongoose.model("Novel", novelSchema);
+// module.exports = mongoose.model("Short", shortsSchema);
+// module.exports = mongoose.model("Article", storiesArticlesSchema);
```

Comment:
It's good practice to not commit commented out code, as it is confusing for others reading it why it's there, and also makes the file more difficult to understand. Are you planning to use these models further down the line? :)

Reply:
Cleaned all commented out code.